Dockerfile

```diff
@@ -16,4 +16,5 @@ FROM nginx:1.23.1 as base
 COPY --from=build /dist /usr/share/nginx/html
 
 # When the container starts, replace the env.js with values from environment variables
-CMD ["/bin/sh",  "-c",  "envsubst < /usr/share/nginx/html/assets/env.template.js > /usr/share/nginx/html/assets/env.js && exec nginx -g 'daemon off;'"]
\ No newline at end of file
+#CMD ["/bin/sh",  "-c",  "envsubst < /usr/share/nginx/html/assets/env.template.js > /usr/share/nginx/html/assets/env.js && exec nginx -g 'daemon off;'"]
+CMD ["/bin/sh",  "-c",  "printenv && exec nginx -g 'daemon off;'"]
```

Comment:
I'll need to pull & test this change later, so long as it doesn't break compatibility with other deployments this seems fine

Reply:
The 'aud' is a required value for obtaining an authorization code, according to 2.1.9.1 in https://build.fhir.org/ig/HL7/smart-app-launch/app-launch.html.

Also, it would be great if I could see the requested scopes, etc, on the SPA. I have been meaning to modify to see the request.. so that I can compare requested scopes from granted scopes. helpful for debugging.

Reply:
I think the issue was I did not have the home-brew envsubst available, so I added printing out the environment to view the env variable.  I rely directly on reading the environment in the js, which I set in docker-compose.